src/main/java/hudson/plugins/sonar/SonarRunnerInstallation.java

```diff
@@ -70,23 +68,27 @@ public SonarRunnerInstallation(String name, String home, List<? extends ToolProp
   }
 
   /**
-  * Gets the executable path of this Sonar runner on the given target system.
-  */
+   * Gets the executable path of this SonarQube Scanner on the given target system.
+   */
   public String getExecutable(Launcher launcher) throws IOException, InterruptedException {
     return launcher.getChannel().call(new MasterToSlaveCallable<String, IOException>() {
       @Override
       public String call() throws IOException {
-        File exe = getExeFile();
+        File exe = getExeFile("sonar-scanner");
         if (exe.exists()) {
           return exe.getPath();
         }
+        File oldExe = getExeFile("sonar-runner");
+        if (oldExe.exists()) {
+          return oldExe.getPath();
+        }
         return null;
       }
     });
   }
 
-  private File getExeFile() {
-    String execName = Functions.isWindows() ? "sonar-runner.bat" : "sonar-runner";
+  private File getExeFile(String name) {
+    String execName = Functions.isWindows() ? name + ".bat" : name;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add parentheses to make the operator precedence explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS864)